### ISSUE TITLE
Build failed due to missing licence spec for lib/commons-text-1.6.jar

### ISSUE THREAD

**Issue by philpem** [NationalSecurityAgency/ghidra#1887]
**Describe the bug**
Cannot build current Ghidra master as of 15th May 2020 due to an error in ip.gradle:

```
No License specified for external library: lib/commons-text-1.6.jar in module /var/lib/jenkins/workspace/Ghidra/ghidra/Ghidra/Framework/Generic
```

**To Reproduce**
Build according to build instructions.

**Expected behavior**
Build completes, output files are produced.

**Environment (please complete the following information):**
 - OS: Linux - Ubuntu 18.04
 - Java Version: openjdk 11.0.7 2020-04-14
 - Ghidra Version: Git master branch HEAD, e08efb1

**Comment by ryanmkurtz** [comment]
Will be fixed by @astrelsky 's #1878 PR.

**Comment by ryanmkurtz** [comment]
Fixed by 1675f226b381e407e24a591d210c6306c1c3fffe.  This issue can be closed.
